common/src/main/java/com/alibaba/nacos/common/notify/NotifyCenter.java

```diff
@@ -46,9 +47,9 @@
     
     private static final Logger LOGGER = LoggerFactory.getLogger(NotifyCenter.class);
     
-    public static int ringBufferSize = 16384;
+    public static int RING_BUFFER_SIZE = 16384;
```

Comment:
No `static final` use camel naming. Your changes can't pass checkstyle

Reply:
I hava alreayd fix this issue in next commit.